salt/etcd-migrate/init.sls

```diff
@@ -0,0 +1,6 @@
+/tmp/create-or-update-etcd-pillar:
```

Comment:
Instead of having a new top level `etcd-migrate` directory, I would move (and this is my personal taste) this file to `etcd/migrate.sls` (or something similar) so we keep everything related to `etcd` in the same place. I would do the same with the `create-or-update-etcd-pillar` script. I've done something similar in my [CNI branch](https://github.com/kubic-project/salt/tree/cni/salt/cni).